### PR TITLE
Add support for view controller based status bar appearance

### PR DESCRIPTION
**Pull request by moritzmoritz**
* I needed to have the status bar in LightContentStyle

Feel free to comment and of course to merge.

### CODE REVIEW

File: Agrume/Agrume.swift [hunk 1]
```diff
@@ -431,3 +436,15 @@ extension Agrume: UICollectionViewDelegate {
   }
 
 }
+
+extension Agrume {
+
+  // MARK: Status Bar
+  public override func preferredStatusBarStyle() -> UIStatusBarStyle {
+    if let statusBarStyle = self.statusBarStyle {
```
Comment (JanGorman) [inline Agrume/Agrume.swift]: simply `return statusBarStyle ?? super.preferredStatusBarStyle()`

File: Agrume/Agrume.swift [hunk 2]
```diff
@@ -31,12 +31,17 @@ public final class Agrume: UIViewController {
...
-    
+
   public typealias DownloadCompletion = (image: UIImage?) -> Void
     
   public var didDismiss: (() -> Void)?
   public var didScroll: ((index: Int) -> Void)?
   public var download: ((url: NSURL, completion: DownloadCompletion) -> Void)?
+  public var statusBarStyle: UIStatusBarStyle? {
+    didSet {
+      self.setNeedsStatusBarAppearanceUpdate()
```
Comment (JanGorman) [inline Agrume/Agrume.swift]: and no need for the `self.` here

**Comment by JanGorman** [comment]
If you could change those two points I'd be happy to merge and cut a new release

File: Agrume/Agrume.swift (hunk 2 above)
Comment (moritzmoritz) [inline Agrume/Agrume.swift]: yeah :+1:

**Comment by moritzmoritz** [comment]
@JanGorman applied feedback :+1:

**Comment by JanGorman** [comment]
@moritzmoritz 🙇 thanks for the PR!

**Pull request merged**